manabi/lock.py

```diff
@@ -258,10 +257,11 @@ def __getitem__(self, token):
             cursor = self._storage_object().execute(
                 "SELECT data FROM manabi_lock WHERE token = %s", (str(token),)
             )
-            lock = cursor.fetchone()
-            if lock is None:
+            locks = cursor.fetchmany(1)
```

Comment:
From the psycopg documentation I gather that if no result set is produced by the execute it will throw a programming error. I'm a bit unsure whether 0 rows will be an issue like with the fetchone and if we need to handle the possible ProgrammingError thrown (see documentation: https://www.psycopg.org/docs/cursor.html#cursor.fetchmany)